tests/base/utils.py

```diff
@@ -212,14 +212,10 @@ def assert_ok_model_acc(trainer, key='test_acc', thr=0.4):
 
 def can_run_gpu_test():
```

Comment:
nit: Maybe rename this function? It does not return a bool anymore (as indicated by the name). Something like `get_num_gpus` should be clearer. 

Also I'm not sure if we really need this function, since we probably don't care on the warnings, but the same is stated in skip reasons.